app/dashboard/templates/dashboard/sidebar_search.html

```diff
@@ -15,7 +15,6 @@
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
 {% load i18n static %}
-<link rel="stylesheet" href={% static "v2/css/sidebar_search.css" %}>
```

Comment:
why remove this css?

Reply:
Can’t embed link elements into vue templates. So I rearranged them.

Reply:
Can’t embed link elements into vue templates. So I rearranged them.